packages/es-dev-server/src/command-line-args.js

```diff
@@ -216,6 +223,11 @@ export function readCommandLineArgs(argv = process.argv, config = {}) {
 
   let { open } = options;
 
+  if ('dedupe' in options) {
+    options.dedupeModules = options.dedupe;
+    delete options.dedupe;
```

Comment:
Just out of interest, why is it needed  to delete `options.dedupe` here?

Reply:
Since I'm aliasing dedupe to dedupeModules, I want to ensure it doesn't accidentally get picked up somewhere else when the config object is passed along.

Reply:
A "cleaner" way would be to just pass a new object to go from cli args to config, but this is less code.

Reply:
A "cleaner" way would be to just pass a new object to go from cli args to config, but this is less code.

Reply:
Cool, makes sense